json_encoding.rst

```diff
@@ -0,0 +1,62 @@
+#########################
+Raiden transport messages
```

Comment:
This document is not describing any message, only the json encoding.

This is best as `JSON encoding`